code/formatters/spdx.py

```diff
@@ -16,6 +16,8 @@
 from json import dumps, loads
 from uuid import uuid4
 from io import StringIO
+from spdx.checksum import Algorithm, sha1, sha512
+import hashlib
```

Comment:
Any functions used from Hashlib should be included in the existing Hashlib import on line 15